SmartDeviceLink/private/SDLMenuManager.m

```diff
@@ -530,6 +526,33 @@ - (BOOL)sdl_isDynamicMenuUpdateActive:(SDLDynamicMenuUpdatesMode)dynamicMenuUpda
     }
 }
 
+- (NSArray<SDLMenuCell *> *)sdl_addUniqueNamesToCells:(nullable NSArray<SDLMenuCell *> *)choices {
```

Comment:
```suggestion
- (void)sdl_addUniqueNamesToCells:(nullable NSArray<SDLMenuCell *> *)choices {
```

1. See choice set manager comment on improving this method
2. I don't think there needs to be a return value and the `subcells` doesn't need to be made public. The array is immutable, but what's within it can change. You don't have to set a new array, you can just modify the cells within the array.